src/Serilog.Sinks.Sentry/SentrySinkExtensions.cs

```diff
@@ -26,15 +27,16 @@ public static class SentrySinkExtensions
             string release = null,
             string environment = null,
             LogEventLevel restrictedToMinimumLevel = LogEventLevel.Error,
-            IFormatProvider formatProvider = null)
+            IFormatProvider formatProvider = null,
+            string tags = null)
```

Comment:
Probably we should use IEnumerable string here?

Reply:
I will check if using `IEnumerable<string>` here works with JSON-based serilog configuration (https://github.com/serilog/serilog-settings-configuration) and change that when successful.

Reply:
Unfortunately, using `IEnumberable<string>` here is not supported when using configuration files and leads to an exception in the configuration reader. The splitup of the string parameter only happens once and shouldn't be a performance issue so until more data types are supported in https://github.com/serilog/serilog-settings-configuration we have to keep parameters simple in order to be able to configure logging in config files instead of hard-coding them.

Reply:
Ah, I see, thank you for the investigation! 👍 

> performance issue

I worry from the design perspective, in strongly typed world `something separated` looks not so good. :)

Well, let's use the comma separated parameter.